products/kirby.md

```diff
@@ -37,7 +45,7 @@ releases:
 -   releaseCycle: "4"
     codename: "A new era"
     releaseDate: 2023-11-28
-    eol: false
+    eol: 2027-06-23
     supportedPhpVersions: 8.1 - 8.3
```

Comment:
```suggestion
    supportedPhpVersions: 8.1 - 8.4
```

Since 4.6 : https://getkirby.com/docs/reference/system/requirements#php-version-support-history.